XenAdmin/Wizards/NewPolicyWizard/NewPolicyWizardSpecific.cs

```diff
@@ -261,8 +261,18 @@ protected override void FinishWizard()
 
         protected override string WizardPaneHelpID()
         {
-            return CurrentStepTabPage is RBACWarningPage ? FormatHelpId("Rbac") : base.WizardPaneHelpID();
+            if (CurrentStepTabPage is RBACWarningPage)
+            {
+                return FormatHelpId("Rbac");
+            }
+
+            else if (!VMGroup<T>.isVMPolicyVMPP)
+            {
+                return "NewPolicyWizardSpecific_" + CurrentStepTabPage.HelpID + "Pane";    
```

Comment:
I propose to use the prefix _NewPolicyWizardVMSS_ instead of _NewPolicyWizardSpecific_, to make it clear that we are refering to the VMSS wizard